packages/main/src/PopoverRegistry.js

```diff
@@ -0,0 +1,112 @@
+import { isClickInRect } from "./PopupUtils.js";
```

Comment:
Please move this file to a directory, try to keep the main only for components themselves

Reply:
I made a directory `popup-utils` name proposals are welcome